src/@next/Text/Text.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Icon } from '../Icon';
+import { Input, InputProps } from '../Input/Input';
+
+export type TextProps = Omit<InputProps, 'type'> & { canClear?: boolean };
+
+export const Text = ({
```

Comment:
I think we should also rename this since Text and Typography are often used interchangeably. maybe we can use `TextField` instead?

Reply:
TextField is actually to use it for `form`, it is another control compose of this. But yeah, let's put Input suffix for all these